src/backend/helpers.js

```diff
@@ -60,16 +60,17 @@ const tagType = {
 };
 
 function mdToHtml(text) {
-    // text = text.replace(/([,;/])(?=\S)/g, '$1\u200b');
-    let result = converter.makeHtml(text).replace(/\n/g, '<br>');
+    if (text === undefined)

```

Comment:
If `text` is null, we could still have issues, right?

Reply:
Yep, but the error is specifically saying its undefined. Ive changed both checks to be nullish checks instead